### PR TITLE
Correct release prepare to include all Optaplanner repos

### PR DESCRIPTION
**Pull request by radtriste**

### CODE REVIEW

File: .ci/jenkins/Jenkinsfile.release.prepare [hunk 1]
```diff
@@ -103,7 +103,7 @@ pipeline {
                         repositories.addAll(collectRepos(RUNTIMES_REPOS, getKogitoReleaseBranch()))
                         repositories.addAll(collectRepos(IMAGES_REPOS, getKogitoReleaseBranch()))
                         repositories.addAll(collectRepos(OPERATOR_REPOS, getKogitoReleaseBranch()))
-                        repositories.addAll(collectRepos([OPTAPLANNER_REPO, OPTAPLANNER_QUICKSTARTS_REPO], getOptaPlannerReleaseBranch()))
+                        repositories.addAll(collectRepos(OPTAPLANNER_REPOS, getOptaPlannerReleaseBranch()))
```
Comment (Ginxo) [inline .ci/jenkins/Jenkinsfile.release.prepare:106]: wouldn't it be...?
```suggestion
                        repositories.addAll(collectRepos([OPTAPLANNER_REPOS], getOptaPlannerReleaseBranch()))
```

**Review by Ginxo — CHANGES_REQUESTED** [review]
please review

File: .ci/jenkins/Jenkinsfile.release.prepare (hunk 1 above)
Comment (radtriste) [reply .ci/jenkins/Jenkinsfile.release.prepare:106]: nope, `OPTAPLANNER_REPOS` is already an array. See https://github.com/kiegroup/kogito-pipelines/blob/22f24e233a85d1e53b9cff12c067dab8f811bb49/.ci/jenkins/Jenkinsfile.release.prepare#L13

**Review by Ginxo — APPROVED** [review]
LGTM, you are right

**Pull request merged**